util/context.go

```diff
@@ -0,0 +1,47 @@
+package util
```

Comment:
[tiny]: can we rename this file `echo_context.go`? Context overlaps a bit with the standard library context package.

Reply:
sure, done, thanks 👍